Makefile

```diff
@@ -40,6 +40,10 @@ origin-tests:
 	@echo "Running origin-tests"
 	ORIGIN_TESTS_FILTER="$(ORIGIN_TESTS_FILTER)" hack/run-origin-tests.sh
 
+mirror-origin-tests:
```

Comment:
can we also have a target that does both?
mirror, then run tests in disocnnected mode from that repo